src/template.html

```diff
@@ -16,11 +16,10 @@
         <div class="heading">Today's To Do</div>
       </div>
       <div class="item">
-        <input type="text" placeholder="Add to your list...">
+        <input class="task-adder-input" type="text" placeholder="Add to your list...">
       </div>
       <div class="item">
         <div class="tasks-container">
-          
         </div>
       </div>
       <div class="item">
```

Comment:
![todolist (copy)](https://user-images.githubusercontent.com/93039359/184033973-8fdf513a-2c19-4973-ad7d-9b69e76a8ffd.jpg)


- [ ] In your design, please insert this add button with functionality to add a new todo in your HTML to follow the design. :blush:

Reply:
@HAMMAS-SALEEM there's no functionality for this button on the video showing the design, so I'm not sure what to do with it.